packages/shadydom/src/patch-prototypes.js

```diff
@@ -57,14 +58,16 @@ const patchMap = {
   Comment: [SlotablePatches],
   CDATASection: [SlotablePatches],
   ProcessingInstruction: [SlotablePatches],
-  Element: [ElementPatches, ParentNodePatches, SlotablePatches,
+  Element: [ElementPatches, ParentNodePatches, ChildNodePatches, SlotablePatches,
     ElementShouldHaveInnerHTML ? ElementOrShadowRootPatches : null,
     !window.HTMLSlotElement ? SlotPatches : null],
   HTMLElement: [HTMLElementPatches, NonStandardHTMLElement],
   HTMLSlotElement: [SlotPatches],
   DocumentFragment: [ParentNodeDocumentOrFragmentPatches, DocumentOrFragmentPatches],
   Document: [DocumentPatches, ParentNodeDocumentOrFragmentPatches, DocumentOrFragmentPatches, DocumentOrShadowRootPatches],
-  Window: [WindowPatches]
+  Window: [WindowPatches],
+  CharacterData: [ChildNodePatches],
+  DocumentType: [ChildNodePatches],
```

Comment:
Maybe it isn't worth trying to make this work on `DocumentType`. Testing it is pretty awkward and the restrictions on where a `DocumentType` can go in a document (only first child of a document, maybe?) and what can go around one (almost nothing except comments and a single following `<html>`?) makes the ChildNode APIs on DocumentType pretty useless.